### PR TITLE
Update angular-bootstrap-ui to 1.1.2

### PR DESCRIPTION
**Pull request by Narretz**

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -11,12 +11,12 @@ function copySrcToBuild {
   rm -rf build/
   mkdir build/
 
-  cp -r src/ build/
+  cp -r src/. build/
```
Comment (gkalpak) [inline build.sh]: What difference does the `.` make ?

File: src/index.html [hunk 2]
```diff
@@ -990,7 +990,7 @@ <h1 id="previous-conferences">Previous Conferences</h1>
   <script src="google-code-prettify/prettify.min.js"></script>
   <script src="js/homepage.js"></script>
   <script src="js/download-data.js"></script>
-  <script src="js/angular-ui-bootstrap.js"></script>
+  <script src="//cdnjs.cloudflare.com/ajax/libs/angular-ui-bootstrap/1.1.2/ui-bootstrap-tpls.min.js"></script>
```
Comment (gkalpak) [inline src/index.html]: The commit message says `v1.2.1` :stuck_out_tongue:

File: build.sh (hunk 1 above)
Comment (Narretz) [inline build.sh]: It indicates that I want to copy the contents of src, not src itself. I assume that must have worked for the build job anyway, but locally it copied the src folder, so I had build/src, which is not what the build.sh expects.

File: src/index.html (hunk 2 above)
Comment (Narretz) [inline src/index.html]: Good thing it's not the othe way round, because 1.2.1 doesn't exist yet :>

File: build.sh (hunk 1 above)
Comment (gkalpak) [inline build.sh]: If you wanted to copy the contents, shouldn't it be `*` instead of `.` ?

Comment (Narretz) [inline build.sh]: I got this from a SO question, and it worked. But yeah, my bash foo is very weak. I'll just move these script changes to another PR, so we can merge the rest safely.

**Comment by petebacondarwin** [comment]
LGTM

**Comment by Narretz** [comment]
Landed as https://github.com/angular/angularjs.org/commit/1206c9971d2d66083fbc0896014ac22b01f5d371

**Pull request closed without merging**